measurements.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	measuredboot "github.com/edgelesssys/uplosi/measured-boot"
+	"github.com/edgelesssys/uplosi/measured-boot/measure"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newMeasurementsCmd() *cobra.Command {
```

Comment:
Please add a section to the readme documenting the command and explaining what is currently does and what not.